components/style/themes/default.less

```diff
@@ -290,6 +290,10 @@
 @table-padding-vertical: 16px;
 @table-padding-horizontal: 8px;
 
+// Tag
+// --
+@tag-default-bg: @background-color-base;
```

Comment:
It is not same color, we prefer to keep it `#f3f3f3` which is more accessible for more devices.